slotify/groups/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from authentication.models import User
+
+class Category(models.Model):
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Tag(models.Model):
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+class Group(models.Model):
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=255)
```

Comment:
This will probably need a unique constraint

Reply:
You mean the group name?

Reply:
btw some of the stuff won't make sense as I just added the CURD for testing purposes so that I can move over to server. Will fix the code later on

Reply:
yep